src/twisted/conch/newsfragments/12046.bugfix

```diff
@@ -0,0 +1 @@
+twisted.conch.insults.window.Widget objects now correctly respond to function keys
```

Comment:
Nope. No change in logic, just fix of state broken for 7 years. Now it works exactly how it was supposed to work (and how it did those 7 years ago, only using `bytes` now instead of `const`). You're free to reword it, though, as english isn't my native language.

Reply:
Thanks. My bad. Sorry for the noise.
Also, sorry for the old review for https://github.com/twisted/twisted/pull/401
It looks like I was the one approving the changes there.

For the release notes maybe

> twisted.conch.insults.window.Widget.functionKeyReceived fix a regression introduced with #8214 in Twisted 16.5.0